jans-cedarling/cedarling/src/bootstrap_config/policy_store_config.rs

```diff
@@ -20,6 +20,10 @@ pub enum PolicyStoreSource {
     ///
     /// The string contains the raw JSON data representing the policy.
     Json(String),
+    /// Read the policy directly from a raw JSON string.
```

Comment:
'JSON' -> 'YAML'

Also, I think it would be a good idea to make a note here that YAML is intended for test fixtures, not for normal operation. (But maybe JSON is the same? I don't know.)

Reply:
fixed in [commit](https://github.com/JanssenProject/jans/pull/10016/commits/69e5b03e1f575aa6d825d180e847cdf05a8f860b)